src/AppRoutes.tsx

```diff
@@ -1,28 +1,40 @@
 import React, { useEffect } from "react";
 import { Switch, Route, RouteProps, Redirect } from "react-router-dom";
-import Home from "./views/home";
-import Profile from "./views/profile";
-import Messages from "./views/messages";
-import Login from "./views/login";
+import Home from "./views/Home";
+import Profile from "./views/Profile";
+import Messages from "./views/Messages";
+import Login from "./features/auth/login/Login";
 import { useDispatch, useSelector } from "react-redux";
 import { RootState } from "./reducers";
 import { authError } from "./features/auth/authSlice";
+import Logout from "./views/Logout";
+
+export const loginRoute = "/login";
+export const loginPasswordRoute = loginRoute + "/password";
```

Comment:
```suggestion
export const loginPasswordRoute = `${loginRoute}/password`;
```
Pretty minor in this case, but template strings are less error prone if you have to glue more things together.